Modules/xSharePoint/DSCResources/MSFT_xSPFarmSolution/MSFT_xSPFarmSolution.schema.mof

```diff
@@ -0,0 +1,26 @@
+/*
+**Description**
+
+This resource will deploy a farm solution to a farm. A version will be stored in the property bag to determin later if the correct version is installed.
```

Comment:
We need these descriptions to be markdown compliant as well because we generate wiki pages with them, so can you update this to start each sentence on a new line?